smac/runhistory/runhistory.py

```diff
@@ -314,7 +323,7 @@ def update_from_json(self, fn, cs):
         """
         new_runhistory = RunHistory(self.aggregate_func)
         new_runhistory.load_json(fn, cs)
-        self.update(runhistory=new_runhistory)
+        self.update(runhistory=new_runhistory, external_data=True)
```

Comment:
>  My "fix" would make the data be used in the EPM building only, right? Didn't SMAC2 treat pSMAC data like this?

Yes, but this was not due to principal considerations but due to problems in the SMAC2 code base (as far as I know). 

> If intensify sees external data, it will take more time for a new configuration to become an incumbent, because the number of evaluated instance seed pairs for the current incumbent grows #workers times faster than usually, possibly making SMAC spend more time in intensify than it would usually do. Is this intended? 

Yes, on the one hand SMAC could need more time to change its incumbent. But on the other hand, the decision will be based on more data. 
Related to this: is it efficient if all pSMAC runs start with the same incumbent --- I'm trying to figure this out right now.
(I can hopefully send you some more details about pSMAC on Wednesday).

> Maybe we should run both versions against each other to see the effect of both?

Maybe yes, but this is rather low on my todo stack right now.